server_admin/topics/roles/composite.adoc

```diff
@@ -2,20 +2,21 @@
 
 === Composite Roles
 
-Any realm or client level role can be turned into a _composite role_.
-A _composite role_ is a role that has one or more additional roles associated with it.
-When a composite role is mapped to the user, the user also gains the roles associated with that composite.  This inheritance
-is recursive so any composite of composites also gets inherited.
+Any realm or client level role can be a _composite role_. A _composite role_ is a role that has one or more additional roles associated with it. When a composite role is mapped to a user, the user also gains the roles associated with that composite.  This inheritance is recursive so any composite of composites also gets inherited.
 
-To turn a regular role into a composite role, go to the role detail page and flip the `Composite Role` switch on.
+To convert a regular role to a composite role:
+. Click the `Roles` left menu item.
+. Click the role to access the roles detail page.
+. Set the `Composite Role` switch to ON.
 
 .Composite Role
 image:{project_images}/composite-role.png[]
 
-Once you flip this switch the role selection UI will be displayed lower on the page and you'll be able to associate
-realm level and client level roles to the composite you are creating.  In this example, the `employee` realm-level
-role was associated with the `developer` composite role.  Any user with the `developer` role will now also inherit
-the `employee` role too.
+The role selection UI is displayed on the page and you can associate realm level and client level roles to the composite you are creating.
```

Comment:
composite role (add role)

Reply:
Can you clarify what the "(add role)" bit means, @andymunro ?

Reply:
Sorry, I mean add the word "role" after "composite"